doc/releases/migration-guide-4.0.rst

```diff
@@ -220,6 +220,11 @@ Networking
   all the network shell activities can be found under ``net`` shell command.
   After this change the bridge shell is used by ``net bridge`` command.
 
+* The Ethernet bridging code is changed to allow similar configuration experience
+  as in Linux. The bridged Ethernet interface can be used normally even if bridging
+  is enabled. The actual bridging is done by a separate virtual network interface that
+  directs network packets to bridged Ethernet interfaces.
```

Comment:
The Bridge API is unstable, so in theory no need to deprecate removed functions, but perhaps for the sake of clarity, maybe we should list here which API functinos were removed/modified?

Reply:
Good point, I will add that.